docs/05-plugins.md

```diff
@@ -147,7 +147,13 @@ Defines a custom loader. Returning `null` defers to other `load` functions (and
 Type: `(options: InputOptions) => InputOptions | null`<br>
 Kind: `sync, sequential`
 
-Reads and replaces or manipulates the options object passed to `rollup.rollup`. Returning `null` does not replace anything. This is the only hook that does not have access to most [plugin context](guide/en#plugin-context) utility functions as it is run before rollup is fully configured.
+Reads and replaces or manipulates the options object passed to `rollup.rollup`. Returning `null` does not replace anything. This hook does not have access to most [plugin context](guide/en#plugin-context) utility functions as it is run before rollup is fully configured.
+
+#### `outputOptions`
+Type: `(outputOptions: OutputOptions) => OutputOptions | null`<br>
+Kind: `sync, sequential`
+
+Reads and replaces or manipulates the output options object passed to `rollup.rollup`. Returning `null` does not replace anything. This hook does not have access to most [plugin context](guide/en#plugin-context) utility functions as it is run before rollup is fully configured.
```

Comment:
Thanks for taking care to update the documentation! Some comments:
- output options are not passed to `rollup.rollup` but to `bundle.generate`
- with my suggestion, the limitations can be removed